src/VisualStudio/Xaml/Impl/Features/DocumentSpan.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.Editor.Xaml.Features
+{
+    internal struct DocumentSpan
```

Comment:
Too bad. I can't use it because it's internal and I can only make internal types visible to VS that are in my assembly.